docs/12-configuration.md

```diff
@@ -21,7 +20,34 @@ running `azb spark cluster create` will create a cluster of 4 Standard\_A2 nodes
 azb spark cluster create --id spark --vm-szie standard_a2 --size 4 --username spark --wait
 ```
 
-
 ## Secrets Configuration
 
-A template file for necessary secrets is given in config/secrets.yaml.template. Copy the file to secrets.yaml and fill in the proper values. See [Getting Started] (./00-getting-started.md) for more information.   
+A template file for necessary secrets is given in config/secrets.yaml.template. After running `azb spark init`, this file will be copied to a .thunderbolt/ directory in your current working directory. Copy or rename the file to .thunderbolt/secrets.yaml and fill in the proper values for your Batch and Storage accounts. See [Getting Started] (./00-getting-started.md) for more information.   
+
+## SSH Configuration
+
+The SSH connection settings can be configured in the ssh.yaml file. Once you have set your desired values in ssh.yaml, you can connect to the master of your cluster using the command `azb spark cluster ssh`. 
+
+For example, with the default ssh cluster configuration:
+```yaml
+# ssh configuration
+
+# cluster_id: <id of the cluster to connect to>
+cluster_id: spark_cluster
+
+# username: <name of the user account to ssh into>
+username: spark
+
+# job_ui_port: <local port where the job ui is forwarded to>
+job_ui_port: 8080
+
+# web_ui_port: <local port where the spark master web ui is forwarded to>
+web_ui_port: 8081
+
+# jupyter_port: <local port which where jupyter is forwarded to>
+jupyter_port: 8082
+```
+
+running the command `azb spark cluster ssh` will attempt to ssh into a cluster called spark_cluster with the username spark. It will forward the job web ui to localhost:8080, the spark master web ui to localhost:8081 and jupyter to localhost:8082.
```

Comment:
some nitpicks:

- "Running..."(capitalize)
- put names like 'spark_cluster' and the username 'spark' in quotations
- Spark Job UI (not job web ui)
- Spark should be capitalized and UI to
- Jupyter should be capitalized

Reply:
Yeah, this was written pretty lazily. Needs to be updated to reflect taking out the cluster_id as a default, too.